dags/pytorch_xla/nightly.py

```diff
@@ -160,12 +160,13 @@ def huggingface():
       ),
       US_CENTRAL2_B,
   ).run()
-  diffusers_v4_8 = task.TpuQueuedResourceTask(
-      test_config.JSonnetTpuVmTest.from_pytorch(
-          "pt-nightly-hf-diffusers-func-v4-8-1vm"
-      ),
-      US_CENTRAL2_B,
-  ).run()
+  # TODO(yeounoh) the dataset is taken down, we need a new SD model with a new dataset.
+  # diffusers_v4_8 = task.TpuQueuedResourceTask(
+  #     test_config.JSonnetTpuVmTest.from_pytorch(
+  #         "pt-nightly-hf-diffusers-func-v4-8-1vm"
+  #     ),
+  #     US_CENTRAL2_B,
+  # ).run()
```

Comment:
also delete the reference at line 172? Similarly for r2_3.py

Reply:
thanks!